### DISCUSSION TITLE
Welcome to godot-play-game-services Discussions!

### DISCUSSION THREAD

**Discussion by Iakobs** [Iakobs/godot-play-game-services#44] [Announcements]
## 👋 Hi everyone!
  I ([Jacob](https://github.com/Iakobs), maintainer of the plugin) am using Discussions as a place to connect with users of the plugin. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other plugin users.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, I would love to know who's using the plugin and how it helped them in their game! comment below with an introduction of yourself and tell me about your project!

**Comment by EthannYakabuski** [comment]
Hey my name is Ethann Y, I'm a software developer by day who works on a web based app that generates automated test cases for our customers and internal users

I spend my evenings doing downhill longboard freeride, creating/playing board games and recently fell in love with godot so im trying my hand at developing some simple games to get my feet wet. 

My only "finished" video game is called Animal Dash - https://github.com/EthannYakabuski/AnimalDash , we're currently in internal testing while I wait for the canada post strike to finish so my license can get delivered and i can verify my google payments profile to set up the final monetization features. hoping for a 2025 Q1 release at this point. 

This plugin definitely saved me hundreds of hours including mainly the brutal pain of deciphering the documentation on googles side, so i very much appreciate Iakobs for this. 

In the meantime, and when i find some extra time ill try to be helpful to any beginner level issues people may have with the plugin - as setting up login/auth/savegames/achievements are all familiar to me now, as well as sharing any other general coding advice ive ran across in my 15+ years programming. can learn about me on my homepage https://piledrivegames.com/  , I am looking forward to interacting with this community and cheers to Iakobs for providing us this plugin. Big Poggers.

**Reply by Iakobs** [reply]
Thanks for sharing, Ethann! I hope you can release your game soon enough!
